src/App.vue

```diff
@@ -1,6 +1,33 @@
 <script setup>
+import { useI18n } from 'vue-i18n'
+
 import { useGlobalGameState } from './gameState'
 
+const { t, locale } = useI18n()
+
+watch(locale, () => {
+  console.log('locale', locale.value)
```

Comment:
remove this

Reply:
Here to trigger the correct content pairing after switching language settings.